src/DynamoCoreWpf/Views/CodeBlocks/CodeBlockEditor.xaml.cs

```diff
@@ -38,11 +38,13 @@ public partial class CodeBlockEditor : UserControl
         public CodeBlockEditor()
         {
             InitializeComponent();
+            WatermarkLabel.Text = Properties.Resources.WatermarkLabelText;
         }
 
         public CodeBlockEditor(NodeView nodeView)
         {
             InitializeComponent();
+            WatermarkLabel.Text = Properties.Resources.WatermarkLabelText;
```

Comment:
Could you just make this constructor call the other one without arguments and just initialize the `WatermarkLabel` in the first constructor?